polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,264 @@
+
+2. **Shared security and scalability** - L1 chains connected to Polkadot benefit from its [Nominated-Proof-of-Stake (NPoS)]( https://wiki.polkadot.network/docs/learn-consensus#nominated-proof-of-stake){target=\_blank} system, providing security out-of-the-box without the need to bootstrap their own
+
+3. **Secure interoperability** - Polkadot's native interoperability enables seamless data and value exchange between parachains. This interoperability can also be used outside of the ecosystem for bridging with external networks
+
+4. **Resilient infrastructure** - decentralized and scalable, Polkadot ensures ongoing support for development and community initiatives via its on-chain [treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank} and governance
+
+5. **Rapid L1 development** - the [Polkadot SDK](link to Polkadot SDK page in polkadot protocol section) allows fast, flexible creation and deployment of Layer-1 chains
+
+6. **Cultivating the next generation of Web3 developers** - Polkadot supports the growth of Web3 core developers through initiatives such as:
+
+    - [Polkadot Blockchain Academy](https://polkadot.com/blockchain-academy){target=\_blank}
+    - [Polkadot Alpha Program](https://polkadot.com/alpha-program){target=\_blank}
+    - [EdX Courses](https://www.edx.org/school/web3x){target=\_blank}
+    - Rust and Substrate Courses (coming soon)
+
+### High Level Architecture
+
+Polkadot features a chain that serves as the central component of the system. This chain is depicted as a ring, encircled by several parachains that are connected to it. 
+
+According to Polkadot's design, any blockchain that can compile to WebAssembly (Wasm) and adheres to the Parachains Protocol become a parachain on the Polkadot network.
+
+Here’s a high-level overview of the Polkadot protocol architecture
+
+![](/images/polkadot-protocol/architecture/polkadot-chain/overview/relay-chain-architecture.webp)
+
+Parachains propose blocks to the Polkadot validators, who perform checks for availability and validity before finalizing them. With the  chain providing security, collators — full nodes of Parachains — can focus on their tasks without needing strong incentives.
+
+The [Cross-Consensus Messaging Format (XCM)](add link to XCM page){target=\_blank} allows Parachains to exchange messages freely, leveraging the  chain's security for trust-free communication.
+
+In order to interact with chains that want to use their own finalization process (e.g. Bitcoin), Polkadot has [bridges](add link to bridges pages, Snowbridge, etc){target=\_blank}  that offer two-way compatibility, meaning that transactions can be made between different parachains.
+
+### Polkadot's Additional Functionalities
+
+The Polkadot  chain oversees crowdloans and auctions. Chain cores were leased through auctions for three-month periods, up to a maximum of two years. 
```

Comment:
Is there something new regarding how we are saying "the Polkadot chain" versus specifying the relay chain?

Reply:
No, it should be the relay chain